<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1DQHH5O" ignored="0" name="Test" tests="3" failures="2" timestamp="14 Mar 2021 04:50:53 GMT" time="18.301" errors="0">
  <testcase name="testloginAPIWithIncorrectCredentials" time="15.969" classname="com.jakartalabs.fullstack_automation.APITests"/>
  <testcase name="testloginAPI" time="16.102" classname="com.jakartalabs.fullstack_automation.APITests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.jakartalabs.utils.APIActions.verifyAPIResponseDetails(APIActions.java:22)
at com.jakartalabs.fullstack_automation.APITests.testloginAPI(APITests.java:27)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testloginAPI -->
  <testcase name="testDashboardAPI" time="1.933" classname="com.jakartalabs.fullstack_automation.APITests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.jakartalabs.utils.APIActions.verifyAPIResponseDetails(APIActions.java:20)
at com.jakartalabs.fullstack_automation.APITests.testDashboardAPI(APITests.java:54)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testDashboardAPI -->
</testsuite> <!-- Test -->
